lib/pool_sup.ex

```diff
@@ -40,6 +40,18 @@ defmodule PoolSup do
   #
   # client API
   #
+  @doc """
+  Returns a specification to start this module under a supervisor.
+  """
+  defun child_spec(args :: list) :: map do
```

Comment:
Can we use `Supervisor.child_spec` as the return type here?

Reply:
Yes, it looks more specific and clarity!
fixed in a4f2c45